src/firefly/js/visualize/task/PlotHipsTask.js

```diff
@@ -254,14 +254,15 @@ export function makeImageOrHiPSAction(rawAction) {
         const hipsRequest= ensureWPR(payload.hipsRequest);
         const imageRequest= ensureWPR(payload.imageRequest);
         const allSkyRequest= ensureWPR(payload.allSkyRequest);
+
         if (!validateHipsAndImage(imageRequest, hipsRequest, payload.fovDegFallOver)) return;
 
 
         const {plotId, fovDegFallOver, pvOptions, attributes, plotAllSkyFirst=false}= payload;
         const viewerId= determineViewerId(payload.viewerId, plotId);
         const size= getSizeInDeg(imageRequest, hipsRequest);
```

Comment:
We talked about if neither specify size then use image could also be false.

Reply:
ok. will fix it.